DeAceroDLabs/marketplace-frontend
# Feature/dmp 836 deliveryAddress and fiscalAddress when creating account

## What is done

[[DMP-836] deliveryAddress and fiscalAddress when creating account](https://deacero.atlassian.net/browse/DMP-836)

## How is done

- DeliveryAddress and fiscalAddress have been separated to make the code easier to read.
- Added internalNumber for DeliveryAddress.
- Solved back button problem with map.

## How to test

1. Run the project via `npm start`.
2. Create a new user, everything should work.
